### DISCUSSION TITLE
Welcome to SkinToneClassifier Discussions!

### DISCUSSION THREAD

**Discussion by ChenglongMa** [ChenglongMa/SkinToneClassifier#8] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Anam-234** [comment]
I've encountered an issue while running the main.py script in the project. The GUI opens successfully, but when I attempt to process an image, an error occurs:
'stone' is not recognized as an internal or external command, operable program, or batch file.
I am running this script in VS Code on a Windows machine.

Here are the steps I followed:

Installed all required dependencies via pip install -r requirements.txt.
Verified that the stone library is installed and available in my environment.
Ran the main.py script, which opens the GUI successfully. However, when trying to process an image, the command execution fails with the error mentioned above.
My assumption is that the script relies on a system-level stone command that is either not available or not properly linked in the environment.

Any guidance on resolving this issue or additional steps I need to configure would be appreciated!

**Reply by ChenglongMa** [reply]
Hi @Anam-234,

Thanks for your feedback. 

As this library has been well designed for a GUI or CLI interface, you don't have to build it from the source code.

You can just run 

```bash
pip install skin-tone-classifier[all] --upgrade
```

to install it.

Then, follow the instructions at [this section](https://github.com/ChenglongMa/SkinToneClassifier?tab=readme-ov-file#use-stone-in-a-gui) to launch a GUI interface.

The basic steps are copied here:

> Steps:
>
> 1. Open the terminal that can run stone (e.g., PowerShell in Windows or Terminal in macOS).
> 2. Type stone (without any parameters) or stone --gui and press Enter to open the GUI.
> 3. Specify the parameters in each tab.
> 4. Click the Start button to start processing the images.

You can also refer to this video tutorial to see how to use it:
https://www.youtube.com/playlist?list=PLYRpHlp-9V_E5ZLhW1hbNaVjS5Zg6b6kQ

Please let me know if you have any other questions.

Thanks!

Best regards,
Chenglong
